django_spanner/base.py

```diff
@@ -111,7 +113,9 @@ def instance(self):
         :rtype: :class:`~google.cloud.spanner_v1.instance.Instance`
         :returns: A new instance owned by the existing Spanner Client.
         """
-        return spanner.Client().instance(self.settings_dict["INSTANCE"])
+        return spanner.Client(
+            project=os.environ["GOOGLE_CLOUD_PROJECT"]
```

Comment:
IIRC the client should automatically use the active gcloud project (`gcloud config get-value project`). I think this change is fine for now, and see that other googleapis libraries do something similar, but it could cause problems if the env var and active gcloud environment don't match.